nullaway/src/main/java/com/uber/nullaway/NullAway.java

```diff
@@ -743,14 +745,18 @@ private Description checkOverriding(
                 + overriddenMethod.toString()
                 + " returns @NonNull";
       }
+      Tree superTree =
```

Comment:
```suggestion
      Tree overriddenMethodTree =
```

Reply:
Changed it. https://github.com/uber/NullAway/pull/517/commits/de094db3a64b4e9d841f07019b2f7566077c2f49